--- a/src/rutabaga_gfx/src/generated/virgl_renderer_bindings.rs
+++ b/src/rutabaga_gfx/src/generated/virgl_renderer_bindings.rs
@@ -380,7 +380,7 @@
 }
 #[cfg(feature = "virgl_resource_map2")]
 extern "C" {
-    pub fn virgl_renderer_resource_map2(
+    pub fn virgl_renderer_resource_map_fixed(
         res_handle: u32,
         map: *const ::std::os::raw::c_void,
         size: u64,
--- a/src/rutabaga_gfx/src/virgl_renderer.rs
+++ b/src/rutabaga_gfx/src/virgl_renderer.rs
@@ -702,7 +702,7 @@
         #[cfg(feature = "virgl_resource_map2")]
         {
             let ret = unsafe {
-                virgl_renderer_resource_map2(
+                virgl_renderer_resource_map_fixed(
                     _resource_id,
                     _addr as *mut libc::c_void,
                     _size,
