diff --git a/src/rutabaga_gfx/src/generated/virgl_renderer_bindings.rs b/src/rutabaga_gfx/src/generated/virgl_renderer_bindings.rs
index 970e500d..84febc30 100644
--- a/src/rutabaga_gfx/src/generated/virgl_renderer_bindings.rs
+++ b/src/rutabaga_gfx/src/generated/virgl_renderer_bindings.rs
@@ -380,7 +380,7 @@ extern "C" {
 }
 #[cfg(feature = "virgl_resource_map2")]
 extern "C" {
-    pub fn virgl_renderer_resource_map2(
+    pub fn virgl_renderer_resource_map_fixed(
         res_handle: u32,
         map: *const ::std::os::raw::c_void,
         size: u64,
diff --git a/src/rutabaga_gfx/src/virgl_renderer.rs b/src/rutabaga_gfx/src/virgl_renderer.rs
index 0fe6249e..ce8a836a 100644
--- a/src/rutabaga_gfx/src/virgl_renderer.rs
+++ b/src/rutabaga_gfx/src/virgl_renderer.rs
@@ -702,7 +702,7 @@ impl RutabagaComponent for VirglRenderer {
         #[cfg(feature = "virgl_resource_map2")]
         {
             let ret = unsafe {
-                virgl_renderer_resource_map2(
+                virgl_renderer_resource_map_fixed(
                     _resource_id,
                     _addr as *mut libc::c_void,
                     _size,
